docs/ops/activation/Tanh_1.md

```diff
@@ -0,0 +1,52 @@
+## Tanh<a name="Tanh"></a> {#openvino_docs_ops_activation_Tanh_1}
+
+**Versioned name**: *Tanh-1*
+
+**Category**: *Activation function*
+
+**Short description**: Tanh activation function.
+
+**Attributes**:
+
+    No attributes available.
+
+**Inputs**:
+
+* **1**: A tensor of type T. **Required.**
+
+**Outputs**:
+
+* **1**: The result of element-wise *Tanh* operation. A tensor of type *T* and the same shape as input tensor.
+**Types**
+
+* *T*: any numeric type.
+
+**Detailed description**
```

Comment:
This section should go under short description section

Reply:
done